tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
Sorry for spamming, but it might be worth trying `fonts.names.resolve_cache()` instead of fontloader.open... just to see if performance improves a little... If we gain 1s per compilation (half a second per font), I think it's very helpful for small documents...